lib/utils/write-to-clipboard.ts

```diff
@@ -0,0 +1,3 @@
+export async function writeToClipboard(url: string) {
+  setTimeout(async () => await navigator.clipboard.writeText(url));
```

Comment:
Didn't realize it was a timing issue. I thought it was a security issue.

Reply:
There's different ways to resolve it. You can use `ClipboardItem` but this breaks in Chrome. I got the simpler solution from here

https://stackoverflow.com/questions/66312944/javascript-clipboard-api-write-does-not-work-in-safari#comment126772784_68241516